src/app/components/elements/CheckLoginOwner.jsx

```diff
@@ -69,14 +69,14 @@ class CheckLoginOwner extends React.Component {
                     <span className="warning uppercase">{tt('g.warning')}:</span> {tt('postfull_jsx.your_password_permissions_were_reduced')} <TimeAgoWrapper date={last_valid_time} />. {tt('postfull_jsx.if_you_did_not_make_this_change') + ' '} <a onClick={this.recover}>{tt('g.recover_your_account')}</a>.
                 </p>
                 <p>
-                    {tt('postfull_jsx.owhership_changed_on')}: {last_valid_date.toString()}
+                    {tt('postfull_jsx.owhership_changed_on', {date: last_valid_date.toString()})}
```

Comment:
this toString... does that localize?  should the date be passed into the tt() function as a date and be allowed to be localized by the translation?